target/linux/ramips/dts/mt7621_yuncore_fap640.dts

```diff
@@ -129,7 +129,7 @@
 					#address-cells = <1>;
 					#size-cells = <1>;
 
-					macaddr_factory_0004: macaddr@0004 {
+					macaddr_factory_0004: macaddr@4 {
 						reg = <0x0004 0x6>;
```

Comment:
(```suggestion
					macaddr_factory_4: macaddr@4 {
						reg = <0x4 0x6>;
```
(and also uses of the label in `&gmac0`)

Reply:
Fixed.